Fix `Show only distinct rows` in tm_data_table (#983)
insightsengineering/teal.modules.general: NEWS.md

- Improve `tm_missing_data` visualization (#495).
- Multiple decorators can be applied to the same output object (#978).

### Bug fixes

> - Fixed `Show only distinct rows` in `tm_data_table` that was incorrectly adding an extra count column `n` (#983).

Reviewer: ```suggestion
- `Show only distinct rows` in `tm_data_table` does no longer show an extra count column `n` (#983).
```
